data/eos-companion-app.service

```diff
@@ -6,7 +6,29 @@ ConditionPathExists=/etc/avahi/services/companion-app.service
 [Service]
 ExecStart=/usr/bin/dbus-run-session /usr/bin/flatpak run --no-desktop --no-a11y-bus --no-documents-portal com.endlessm.CompanionAppService
 User=companion-app-helper
+# This does not appear to cause any problems when writing
+# to /var/lib/eos-companion-app, remove if it does. We need
+# CAP_SETUID and CAP_SETGID as bwrap is setuid
```

Comment:
Nitpick: Would be good to briefly motivate the inclusion of `CAP_NET_ADMIN`, `CAP_SYS_ADMIN` and `CAP_SYS_CHROOT` too.

Reply:
Done